docs/traffic-policy/macros/security.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Security Macros
+description: Security Macros allow you to access sensitive information directly in your Traffic Policies. 
+---
+
+Security Macros allow you to access sensitive information directly in your Traffic Policies. Your ngrok account has a Vault that can store Secrets. Any secrets that you add to your vault will be available across your account on all traffic policies. Updates to these secrets will be reflected across all traffic policies automatically.
+
+## How secrets are secured
+
+- Secrets are protected at rest using industry standard `AES-256` encryption
+- ngrok's REST API does not return secrets as part of any of its response payloads
+- REST API traffic is encrypted in-transit using `HTTP/S` and `TLS 1.2+`
+
+## Using secrets in you Kubernetes deployment
+
+TBD
+
+## Macros
+
+### `secret(string, string) -> string`
+
+Takes the vault name as the first argument and the secret name as the second argument. Returns the secret value.
+
```

Comment:
We should also mention which actions are supported while in Early Access
- `set-vars`
  - CEL is supported in the following fields:
    - `vars`
- `verify-webhook`
    - CEL is supported in the following fields:
      - `secret`
- `basic-auth`
   - CEL is supported in the following fields:
     - `credentials`
- `jwt-validation`
  - CEL is supported in the following fields:
    - `issuer.allow_list[*].value`
    - `http.tokens[*].name`
    - `http.tokens[*].prefix`
    - `jws.keys[*].sources[*].additional_jkus`